components/card/src/css/vertical-card.css

```diff
@@ -0,0 +1,120 @@
+@import url(../../../../css/component.css);
+.jersey-container {
+  position: relative;
+  width: 320px;
+  height: 420px;
+  background: var(--jersey-primary-color);
+  border-radius: 20px;
+  overflow: hidden;
+}
+
+.jersey-container::before {
+  content: "";
+  position: absolute;
+  top: -50%;
+  width: 100%;
+  height: 100%;
+  background: var(--jerysey-secondary-color);
+  transform: skewY(345deg);
+  transition: 0.5s;
+}
+
+.jersey-container:hover::before {
+  top: -70%;
+  transform: skewY(390deg);
+}
+
+/* To Display Watermark */
+
+/* .jersey-container::after {
+  content: "MILAN";
```

Comment:
remove the commented code if it's not for future use

Reply:
I am going to use it in future